api/store/mongo/utils.go

```diff
@@ -85,6 +85,8 @@ func buildFilterQuery(filters []models.Filter) ([]bson.M, error) {
 				}
 
 				property = bson.M{"$gt": value}
+			case "all":
```

Comment:
As `all` has a special meaning, we need to forbid a tag named _all_ to be added.